### PR TITLE
Handle crypt error

### PR DESCRIPTION
**Pull request by jochenschneider**

### CODE REVIEW

**Comment by jochenschneider** [comment]
What is the status of this?

File: ota-plus-web/test/com/advancedtelematic/controllers/ClientToolControllerSpec.scala [hunk 1]
```diff
@@ -160,5 +171,19 @@ class ClientToolControllerSpec extends PlaySpec with GuiceOneServerPerSuite with
...
+
+    "error when when crypt client can't connect to crypt server" in {
+      val application = builder.overrides(bind[WSClient].to(mockClientFailingCrypt)).build
+      val controller = application.injector.instanceOf[ClientToolController]
+
+      try {
+        status(call(controller.downloadClientToolBundle(UUID.randomUUID()), request))
+        throw new Exception("shouldn't happen")
+      } catch {
+        case _: RemoteApiError =>
```
Comment (stevana) [inline ota-plus-web/test/com/advancedtelematic/controllers/ClientToolControllerSpec.scala]: Empty right hand side? What does that mean?

Comment (jochenschneider) [reply ota-plus-web/test/com/advancedtelematic/controllers/ClientToolControllerSpec.scala]: It means to ignore this particular exception.

Comment (jochenschneider) [reply ota-plus-web/test/com/advancedtelematic/controllers/ClientToolControllerSpec.scala]: So here it means: test that controller.downloadClientToolBundle() throws an RemoteApiError.

**Comment by jochenschneider** [comment]
Updated to use Scalatest DSL to check for exception.

**Pull request merged**